### PR TITLE
added PINIO config for IF F7 PRO

### PR DESCRIPTION
**Pull request by Guidus93**
Added PINIO configuration

### CODE REVIEW

File: configs/default/IFRC-IFLIGHT_BLITZ_F7_PRO.config [hunk 1]
```diff
@@ -98,7 +102,7 @@ set mag_bustype = I2C
 set mag_i2c_device = 2
 set baro_bustype = I2C
 set baro_i2c_device = 2
-set serialrx_provider = CRSF
+set serialrx_provider = CRSF
```
Comment (blckmn) [inline configs/default/IFRC-IFLIGHT_BLITZ_F7_PRO.config:105]: whitespace?

Comment (Guidus93) [reply configs/default/IFRC-IFLIGHT_BLITZ_F7_PRO.config]: fixed, sorry my bad

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
@Guidus93 

Please also add #defines for the target as we are migrating to a new build API.

For an example please look at https://github.com/betaflight/unified-targets/blob/master/configs/default/AIRB-NOX.config

**Comment by Guidus93** [comment]
> @Guidus93
> 
> Please also add #defines for the target as we are migrating to a new build API.
> 
> For an example please look at https://github.com/betaflight/unified-targets/blob/master/configs/default/AIRB-NOX.config

Done

**Review by KarateBrot — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**